hw_4/script.js

```diff
@@ -0,0 +1,25 @@
+function factorial(n) {
+  return n <= 1 ? 1 : n * factorial(n - 1);
+}
+
+var arr1 = [];
```

Comment:
Why you use here `var` insted `let ` and `arr1` insted simple `arr`?

Reply:
https://github.com/BlackTruth/javascript/blob/4d02cb8e9b7adf484665c236844a5d36591978a6/hw_4/script.js#L5

FIxed.